olive/model/__init__.py

```diff
@@ -301,7 +299,8 @@ def get_onnx_file_path(model_path: str, onnx_file_name: Optional[str] = None) ->
     @property
     def model_path(self) -> str:
         model_path = super().model_path
-        model_path = self.get_onnx_file_path(model_path, self.onnx_file_name) if model_path else None
+        if model_path and Path(model_path).is_dir():
```

Comment:
This part doesn't actually need to be changed. `self.get_onnx_file_path` already handles the `file` case and also ensures that `model_path` exists.

Reply:
just see https://github.com/microsoft/Olive/commit/1f8427642e06c2927ba9bd35561faf0cbf2f81f7#diff-2e0c7ca9e36053f5b553ea13c7c506d26abced76ff0efd0b9f8b85043972cce3R322 has this check

Reply:
https://github.com/microsoft/Olive/blob/main/olive/model/__init__.py#L263

Reply:
actually, I am a little bit confused about the condition is not same originally when call get_onnx_file_path

Reply:
Yeah, iirc, when I added it, I wanted to make sure the local folder had the .onnx file inside it. But looking at it now, there was no need to distinguish between local file and local folder since `self.get_onnx_file_path` handles both cases. 

Not really an issue. Just makes the logic cleaner.

Reply:
will remove it